### PR TITLE
fix ErrorString func issue

### PR DESCRIPTION
**Pull request by yxxhero**
Introduced in #99

### CODE REVIEW

**Review by elezar — CHANGES_REQUESTED** [review]

File: gen/nvml/return.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ import (
 
 // nvml.ErrorString()
 func ErrorString(r Return) string {
-	if err := GetLibrary().Lookup("nvmlErrorString"); err != nil {
+	if err := GetLibrary().Lookup("nvmlErrorString"); err == nil {
```
[the remark this answers is not in the document]
Comment (yxxhero) [reply gen/nvml/return.go:23]: @elezar ok. got it.

**Comment by yxxhero** [comment]
@elezar done

**Comment by yxxhero** [comment]
@elezar thanks for your advice.

**Review by elezar — APPROVED** [review]
Thanks for fixing this! 

LGTM.

**Pull request merged**